packages/palette/src/elements/Button/Button.tsx

```diff
@@ -195,10 +196,12 @@ export class ButtonBase extends Component<ButtonBaseProps & SansProps> {
 
     return (
       <Container
-        {...rest}
-        className={[loadingClass, disabledClass].join(" ")}
+        className={[loadingClass, disabledClass, className]
```

Comment:
Yeah, no clue now.... I see that we're using props [here](https://github.com/artsy/palette/blob/9e98e47615c0197db2ea980586097dae196abe29/packages/palette/src/elements/Button/Button.tsx#L262) but not too sure about the reasoning behind this line.